libs/community/langchain_community/chains/pebblo_retrieval/base.py

```diff
@@ -326,6 +327,7 @@ def _get_app_details(  # type: ignore
             framework=framework,
             chains=chains,
             plugin_version=PLUGIN_VERSION,
+            langchain_community_version=langchain_community.__version__,
```

Comment:
We should make this field framework neutral. Perhaps we can reuse `plugin_version`? The value it takes could be `langchain-community x.y.z`, `llamaindex x.y.z`, etc.

Reply:
@dristysrivastava We can keep `plugin_version` for backward compatibility, deprecate and remove it in future releases. However, given we will support a wide variety of client types (overtime) like `langchain`, `llamaindex`, `haystack`, `autogen` .. and ETL partners, it makes sense to keep this neutral.

Reply:
Updated it to generic field, here as well as in pebblo.